README.md

```diff
@@ -12,36 +12,36 @@ Once you have cloned this repo, install the necessary dependencies:
 npm i
 ```
 ### Set up your `.env` file
-Create a file called `.env` in the root folder of the project and copy the contents of `sample.env` into that new file
+Create a file called `.env` in the root folder of the project.
 
-### Run local database with docker compose
+If using a local database, copy the contents of `sample.env` into that new file. If using the remote database, use our password manager to find credentials for the dev database.
+
+### (Optional) Use a local database
```

Comment:
This phrasing is fine for a first step. But, I think it has implications on how AE manages database environments. This makes it worth spinning out into a new issue for discussion and refined documentation. See #199